jme3-android-examples/jme3-android-tests/jbullet/build/.transforms/fc8b320a5f3dbdbd90fc10b5fc27ef4c.bin

```diff
@@ -0,0 +1 @@
+o/jetified-jbullet
```

Comment:
what is this .bin and `jetified-jbullet/classes.dex`?

Reply:
yep, as i answered before, they are build files for jbullet.jar files, since they are imported as modules, they get extracted,& build by android